src/Symfony/Component/Console/Logger/ConsoleLogger.php

```diff
@@ -113,4 +117,15 @@ private function interpolate($message, array $context)
         // interpolate replacement values into the message and return
         return strtr($message, $replace);
     }
+
+    /**
+     * @param $formatLevel
+     * @param $level
+     * @param $message
+     * @return string
+     */
+    protected function formatOutput($formatLevel, $level, $message)
```

Comment:
I'm not a big fan of extension points being protected methods. What about just making `<%1$s>[%2$s] %3$s</%1$s>` configurable instead by adding it as an argument to the constructor? That should be enough.